trio/_dtls.py

```diff
@@ -784,25 +839,27 @@ def __init__(self, endpoint, peer_address, ctx):
         # OP_NO_RENEGOTIATION disables renegotiation, which is too complex for us to
         # support and isn't useful anyway -- especially for DTLS where it's equivalent
         # to just performing a new handshake.
-        ctx.set_options(SSL.OP_NO_QUERY_MTU | SSL.OP_NO_RENEGOTIATION)
+        ctx.set_options(SSL.OP_NO_QUERY_MTU | SSL.OP_NO_RENEGOTIATION)  # type: ignore[attr-defined]
```

Comment:
Is this `attr-defined` for `ctx.set_options` or `SSL.X`?

Reply:
`SSL.OP_NO_RENEGOTIATION`. Coerced black into splitting across a couple lines to make it apparent which one it's referring to. No clue why it's being complained about or if it's an issue.